CFUValidator/CommandLineOptions/CommandLineParser.cs

```diff
@@ -46,7 +46,9 @@ public class CommandLineParser
             new CommandLineArgument(OptionUsage.Url, 'u', "URL of CFU XML. Cannot be used with -f flag."),
             new CommandLineArgument(OptionUsage.File, 'f', "Path to CFU XML file. Cannot be used with -u flag."),
             new CommandLineArgument(OptionUsage.ServerVersion, 's', "Server version to test, eg. 6.0.2."),
-            new CommandLineArgument(OptionUsage.Hotfix, 'p', "Space delimited list of patches/hotfixes to test, e.g. XS602E001.")
+            new CommandLineArgument(OptionUsage.Hotfix, 'p', "Space delimited list of patches/hotfixes to test, e.g. XS602E001."),
+            new CommandLineArgument(OptionUsage.Username, 'n', "Username for downloading hotfixes, e.g. john123"),
+            new CommandLineArgument(OptionUsage.ClientId, 'k', "ClientId to use for downloading hotfixes, eg. das1djs43ahgd5jgjh0")

```

Comment:
This is very small but SonarQube might complain, thinking this is an actual password. I'd use something less "realistic", or remove the example altogether